sdk/tables/azure-data-tables/tests/test_table_async.py

```diff
@@ -253,22 +250,21 @@ async def test_list_tables_with_marker(self, resource_group, location, storage_a
             await self._create_table(ts, prefix + str(i), table_names)
 
         # Act
-        generator1 = ts.list_tables(query_options=QueryOptions(top=2)).by_page()
+        generator1 = ts.list_tables(results_per_page=2).by_page()
         tables1 = []
-        async for el in await generator1: #.__anext__():
+        for el in generator1:
```

Comment:
Why are we no longer using async loops here?

Reply:
Fixed!